controllers/osccluster_loadbalancer_controller_unit_test.go

```diff
@@ -511,11 +516,28 @@ func TestReconcileLoadBalancer(t *testing.T) {
 			expSecurityGroupFound:         true,
 			expCreateLoadBalancerFound:    true,
 			expConfigureLoadBalancerFound: true,
+			expDeleteOutboudSgRule:        true,
+			expDeleteOutboudSgRuleErr:     nil,
 			expDescribeLoadBalancerErr:    nil,
 			expCreateLoadBalancerErr:      nil,
 			expConfigureLoadBalancerErr:   nil,
 			expReconcileLoadBalancerErr:   nil,
 		},
+		{
+			name:                          "failed to empty outboud Sg for loadBalancer",
+			spec:                          defaultLoadBalancerInitialize,
+			expLoadBalancerFound:          false,
+			expSubnetFound:                true,
+			expSecurityGroupFound:         true,
+			expCreateLoadBalancerFound:    true,
+			expConfigureLoadBalancerFound: false,
+			expDeleteOutboudSgRule:        false,
+			expDeleteOutboudSgRuleErr:     fmt.Errorf("DeleteSecurityGroupsRules generic error"),
+			expDescribeLoadBalancerErr:    nil,
+			expCreateLoadBalancerErr:      nil,
+			expConfigureLoadBalancerErr:   nil,
+			expReconcileLoadBalancerErr:   fmt.Errorf("DeleteSecurityGroupsRules generic error cannot empty Outboud sg rules for loadBalancer for Osccluster test-system/test-osc"),
```

Comment:
Can not

Reply:
Like other:
"ConfigureLoadBalancer generic error Can not configure healthcheck for Osccluster test-system/test-osc"